infra/relayer/migrations/20190620211348-enum-add-gaslimit.js

```diff
@@ -0,0 +1,13 @@
+'use strict'
+
+const ENUM_NAME = 'enum_relayer_txn_status'
+
+module.exports = {
+  up: (queryInterface) => {
+    return queryInterface.sequelize.query(`ALTER TYPE ${ENUM_NAME} ADD VALUE 'GasLimit';`)
+  },
+
+  down: () => {
```

Comment:
Don't we need to return a Promise ? Maybe return Promise.resolve()

Reply:
Not sure what you mean?  This has been tested to work, btw.

Reply:
Did you try to run sequelize db:migrate:down ? If yes, I could be wrong - I thought I encountered an issue on similar code in a PR of mine...

Reply:
I did not realize you were talking about `down()` for some reason.  You are correct, I get this with an `undo`:

    ERROR: Migration 20190620211348-enum-add-gaslimit.js (or wrapper) didn't return a promise

Will get a PR up shortly